linkerd/app/src/env.rs

```diff
@@ -283,8 +283,12 @@ pub fn parse_config<S: Strings>(strings: &S) -> Result<super::Config, EnvError>
 
     let hostname = strings.get(ENV_HOSTNAME);
 
-    let f = fs::read_to_string(LABELS_FILE_PATH);
-    let labels = f.ok().map(convert_labels_to_map);
+    let labels = strings.get(ENV_LABELS_FILE_PATH).map(|path_option| {
+        path_option
+            .map(fs::read_to_string)
```

Comment:
TIOLI: do you think we should log here if the path doesn't exist? Since the file not existing and the file containing invalid contents are different errors, it could be useful to record which one happened for debugging purposes.

Reply:
Yep, but should we err out? I added `warn` for most cases (It's a bit verbose though :p ) as labels should not be necessary.